docs/removing-documents.md

```diff
@@ -1,33 +1,15 @@
-# Withdrawing and removing documents
+# Remove / remove and redirect documents
 
 We need to allow users to unpublish content from GOV.UK. In Content Publisher we are replacing
 unpublish terminology with "withdrawn" and "removed".
```

Comment:
I think we need a bit more explanation about withdrawn, if we're mentioning it here. But I'm not really sure what the point of this file is - do we need it?

Reply:
Not sure we do, but as we can't remove a document through the UI it's maybe best we keep it for guidance for people dealing with support requests maybe?

Reply:
I'll just remove the mention of 'withdrawn' as it isn't relevant to the rest of the doc